index.js

```diff
@@ -205,7 +211,11 @@ var TruffleContractSchema = {
     // update schema version
     normalized.schemaVersion = pkgVersion;
 
-    return normalized;
+    if (options.validate) {
```

Comment:
Yep, Truffle's tests.

Oh, I remember now. The problem was with link references not conforming to the regular expression. Seemed like a bit too much trouble to fix right away, because it'd require deciding to include link references as part of the schema (and I know you have plans for those)